src/Agent.Worker/TaskManager.cs

```diff
@@ -135,7 +135,6 @@ public Definition Load(TaskReference task)
             definition.Data = JsonConvert.DeserializeObject<DefinitionData>(json);
 
             // Replace the macros within the handler data sections.
-            // TODO: Do other macros need to be replaced within the handler section? Currently only the $(currentdirectory) macro is handled. Or does the handler need to deal with that.
```

Comment:
I went ahead and removed this TODO. I *think* it's safe to assume we don't care.